removeFromArray/removeFromArray.js

```diff
@@ -2,4 +2,31 @@ const removeFromArray = function() {
 
 }
 
+/* Try without using forEach()
+
+const removeFromArray = function(array, ...toRemove) {
+    let testArray = [the array to be passed in];
+    let newArray = [];
+    for (let i = 0; i < array.length; i++) {
+        if (testArray[i] !== toRemove) {
+            newArray.push(item);
+        }
+        return newArray;
```

Comment:
The indentation here is not correct. This return statement is inside the `for` loop, which means it's going to immediately return the `newArray` after processing the first element. Only return once you've iterated over all the elements in the test array, which means you need the return outside of the loop.